libraries/AP_BattMonitor/AP_BattMonitor.cpp

```diff
@@ -265,6 +269,17 @@ float AP_BattMonitor::voltage(uint8_t instance) const
     }
 }
 
+/// get voltage with sag removed (based on battery current draw and resistance)
```

Comment:
Yes, it's like @OXINARF says.  It's a check that the compensated voltage is reasonable.  I'll modify the comments in the .h and .cpp file to make it clear that "this will always be greater than or equal to the raw voltage"